app/renderer/js/pages/preference/base-section.js

```diff
@@ -18,7 +18,7 @@ class BaseSection extends BaseComponent {
 	}
 
 	generateOptionTemplate(settingOption, disabled) {
-		const label = disabled ? `<label class="disallowed" />` : `<label/>`;
+		const label = disabled ? `<label class="disallowed" title="Setting locked by admin."/>` : `<label/>`;
```

Comment:
@timabbott, thoughts on what this string should be? I like the "Setting locked by X" format, but not sure what to call the person doing the locking here.

Reply:
I think "Setting locked by your system administrator." is probably correct, since "system administrator" pretty unambiguously refers to folks with root access on your computer.

Reply:
yeah, I like that.